ledger/metrics/src/main/scala/com/daml/metrics/MetricsReporting.scala

```diff
@@ -45,7 +42,7 @@ final class MetricsReporting(
         .map(_.start())
       _ <- extraMetricsReporter.fold(Resource.unit) { reporter =>
         acquire(reporter.register(registry))
-          .map(_.start(extraMetricsReportingInterval.getSeconds, TimeUnit.SECONDS))
+          .map(_.start(extraMetricsReportingInterval.toNanos, TimeUnit.NANOSECONDS))
```

Comment:
Nanoseconds are _way_ too fine-grained. Seconds is sufficient.

Reply:
I agree that it is quite fine grained but is it so bad to have it that like this? Seconds are in my opinion not fine grained enough, would down to Milliseconds be fine? Then the durations can at least be a bit finer than seconds (e.g. 2.5 seconds).

Reply:
This is not the duration though its the reporting interval afaict?

Reply:
correct, here the input from the cli is handed over to the reporter as interval.

Reply:
Then I don’t understand why you’re worried about subsecond granularity. Of course the indvidual metrics should be able to handle durations that are smaller than a second but reporting more frequently than once per second doesn’t seem useful.